src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -186,7 +187,7 @@ export default class AxisClipSliders extends React.Component<AxisClipSlidersProp
 
   // When user finishes moving the active slider, update slice label
   makeSliderSetFn(axis: string) {
-    return (values: number[]) => this.setSliderState(axis, values);
+    return (values: number[]) => this.setSliderState(axis, [values[0], values[values.length - 1]]);
```

Comment:
a comment about handling single slider vs range slider might be in order here also